pkg/asset/agent/manifests/agentclusterinstall.go

```diff
@@ -156,8 +157,8 @@ func (a *AgentClusterInstall) Generate(dependencies asset.Parents) error {
 			},
```

Comment:
Based on the latest version of https://github.com/openshift/assisted-service/pull/5438, we also need to set `agentClusterInstall.Spec.External.PlatformName = installConfig.Config.Platform.External.PlatformName` somewhere in `Generate()`.

Reply:
Added https://github.com/openshift/installer/pull/7442/files#diff-8c25c4539e452f6075222c27d475ff8586f3843ea64d79ace077c3a3062b3f8aR160-R165